src/Jackett.Common/Definitions/exttorrents.yml

```diff
@@ -106,8 +106,14 @@ download:
 
 search:
   paths:
-    - path: "{{ if .Keywords }}search/?q={{ .Keywords }}&order={{ .Config.sort }}&sort={{ .Config.type }}{{ else }}latest/{{ end }}"
-    - path: "{{ if .Keywords }}search/?q={{ .Keywords }}/2/&order={{ .Config.sort }}&sort={{ .Config.type }}{{ else }}latest/2/{{ end }}"
+    - path: "{{ if .Keywords }}search/{{ .Keywords }}/?order={{ .Config.sort }}&sort={{ .Config.type }}{{ else }}latest/{{ end }}"
+    - path: "{{ if .Keywords }}search/{{ .Keywords }}/2/?order={{ .Config.sort }}&sort={{ .Config.type }}{{ else }}latest/2/{{ end }}"
```

Comment:
Um, where did you get that path from?
the web site generates `https://ext.to/search/?order=age&sort=desc&q=man+called+otto`

Reply:
Oh, I C. the second page turns into `https://ext.to/search/man-called-otto/2/?order=age&sort=desc`